core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/client/transactional/TransactionManager.java

```diff
@@ -2996,6 +3022,11 @@ else if (tableOption.equalsIgnoreCase("USE_DEFAULT"))
                    (Long.parseLong(tableOption));
                returnStatus.setTableDescriptorChanged();
                break ;
+           case HBASE_HDFS_STORAGE_POLICY:
```

Comment:
General question.  Is it too late to change (alter ) storage policy of a table after it is created. Should this is part of create?